docs/changelog.md

```diff
@@ -10,13 +10,18 @@ We use _breaking :warning:_ to mark changes that are not backward compatible \(r
 
 ## Unreleased
 
-* [\#240](https://github.com/tellor-io/telliot/issues/240) Replaced the cli package to allow for command specific flags and configuration. Now all flags should be passed last. Example:
+## [v5.6.0](https://github.com/tellor-io/telliot/releases/tag/v5.6.0) - 2021.02.08
 
+### Changed
+* [\#240](https://github.com/tellor-io/telliot/issues/240) Replaced the cli package to allow for command specific flags and configuration. Now all flags should be passed last. Example:
   Instead of: `./telliot --config="config.json" stake deposit`, it becomes:`./telliot stake deposit --config="config.json"`
-
   Removed the `RemoteMining` `-r` flag. Remote is active when specifying a `RemoteDBHost` for the `Mine` command. See the `configs/config.json` for an example.
 
-### Changed
+* Removed the features of GPU mining connection to Stratum Pool, as neither were being used.[\#378](https://github.com/tellor-io/telliot/pull/378),[\#39-](https://github.com/tellor-io/telliot/pull/390) and [\#397](https://github.com/tellor-io/telliot/pull/397)
```

Comment:
I think let keeo these 2 separate entries - GPU mining removal and Stratum connection removal. Also think we can ignore 397 as this is not important for people using telliot.

Also could you keep the same formating to be consistent. The PR number and link in front like the other entries.